upgrade-tests/src/test/java/io/zeebe/test/UpgradeTestCase.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.test;
+
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.model.bpmn.BpmnModelInstance;
+import io.zeebe.util.collection.Tuple;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+final class UpgradeTestCase {
```

Comment:
Is there a reason why you don't use visibility modifiers (private/public) anymore?

Reply:
This class is pretty much tied to the upgrade test in the same package so I didn't think it was important since it's unlikely other tests will use it. But I can change it